### PR TITLE
Add support dolphin

### PR DESCRIPTION
**Pull request by ericpre**
- [x] Convert to f-string (drop support for python <3.6),
- [x] add support for Dolphin file manager,
- [x] ready for review.

### CODE REVIEW

**Comment by ericpre** [comment]
Close #1.

**Comment by francisco-dlp** [comment]
The code looks sound, but I don't have KDE to test it. Maybe somebody else (@jat255 ?) could have a look?

**Comment by jat255** [comment]
LGTM. I tested it on my Dolphin installation and it worked as described. Perhaps a note in the README that for Dolphin the commands go into the "Actions" menu would be appropriate (maybe with a screenshot). You could use the screenshot from my (very) out of date PR, if you want, or happy to take a new one:

image

File: start_jupyter_cm/linux.py [hunk 1]
```diff
@@ -19,18 +22,56 @@
 folders = [path for path in sys.argv[1:] if os.path.isdir(path)]
 any_file_selected = len(folders) < len(sys.argv[1:])
 if any_file_selected:
-    subprocess.Popen(["%s/jupyter-%s"])
+    subprocess.Popen(["{path}/jupyter-{terminal}"])
```
Comment (jat255) [inline start_jupyter_cm/linux.py:25]: Is this missing and "f" before the string, i.e. `f"{path}/jupyter-{terminal}"`, or is this a syntax shortcut that I'm not aware of?

File: start_jupyter_cm/linux.py [hunk 2]
```diff
@@ -19,18 +22,56 @@
...
 for folder in folders:
     os.chdir(folder)
-    subprocess.Popen(["%s/jupyter-%s"])
+    subprocess.Popen(["{path}/jupyter-{terminal}"])
```
Comment (jat255) [inline start_jupyter_cm/linux.py:28]: Same as L25

File: start_jupyter_cm/linux.py (hunk 1 above)
Comment (jat255) [reply start_jupyter_cm/linux.py:25]: nvm, I see this is part of an `f` triple quote string... my bad. Stupid Github auto-hide...

**Comment by ericpre** [comment]
Thanks for the review, I added a comment on where the menu will appear. I feel there are already enough images in the readme.rst file (I have tried to make the gnome image smaller) and for this reason I didn't take the screenshot of the previous PR.

**Comment by jat255** [comment]
Ok. LGTM

File: README.rst [hunk 3]
```diff
@@ -21,60 +21,64 @@ in a folder. Currently it only supports Microsoft Windows, GNOME (and
...
+
+On linux, the supported file managers are: Nautilus, Caja and Dolphin. With
+Nautilus and Caja, the shortcut will appear in the *Scripts* menu and with
+Dolphin, it will appear in the *Actions* menu.
 
-.. figure:: images/jupyter_cm_gnome.png
-   :alt: Jupyter context menu entries in gnome
+.. figure:: https://github.com/hyperspy/start_jupyter_cm/raw/master/images/jupyter_cm_gnome.png
+   :alt: Jupyter context menu entries in Nautilus
+   :width: 400px
```
Comment (ericpre) [inline README.rst:48]: The size is ignored on github and this seems to be known bug: https://github.com/github/markup/issues/295, maybe worth using html...

**Pull request merged**